packages/app/src/containers/Canvas.tsx

```diff
@@ -71,6 +74,12 @@ function getPoint(event: MouseEvent): Point {
   return { x: Math.round(event.offsetX), y: Math.round(event.offsetY) };
 }
 
+function isMoving(point: Point, origin: Point): boolean {
+  if (Math.abs(point.x - origin.x) > 2 || Math.abs(point.y - origin.y) > 2) {
```

Comment:
In cases like this, with a single if/else that returns a bool, we can usually return the expression directly. E.g. `return Math.abs(...) || ...`